joomla/joomla-cms
# Make sure only non blocked users can recive actionlogs emails

### Summary of Changes

Make sure only non blocked users can reciver actionlogs emails

### Testing Instructions

create an user
Setup actionlogs notifications for that user
trigger notifications
disable / block the user
trigger more notifications
notice that the now blocked user does not get the notifications any more

### Actual result BEFORE applying this Pull Request

Even blocked users get notifications 

### Expected result AFTER applying this Pull Request

Blocked users does not get notifications anymore

### Documentation Changes Required

None
